test/gui/tst_addAccount/test.feature

```diff
@@ -1,3 +1,4 @@
+@skip @issue-9733
```

Comment:
So we abandon the 2.10 branch from now?

Reply:
We do not abandon it completely. It is of less priority from now on.